app/Filament/Resources/NewsResource/Actions/ProcessUploadedImageAction.php

```diff
@@ -41,6 +41,18 @@ public function execute(string $tempImagePath, ImageUploadType $imageUploadType)
                     $file = createFileArrayFromDataUrl($dataUrl);
                     $imagePath = UploadGameImage($file, ImageType::GameIcon);
                     break;
+                case ImageUploadType::GameBoxArt:
+                    $file = createFileArrayFromDataUrl($dataUrl);
+                    $imagePath = UploadGameImage($file, ImageType::GameBoxArt);
+                    break;
+                case ImageUploadType::GameTitle:
+                    $file = createFileArrayFromDataUrl($dataUrl);
+                    $imagePath = UploadGameImage($file, ImageType::GameTitle);
+                    break;
+                case ImageUploadType::GameInGame:
+                    $file = createFileArrayFromDataUrl($dataUrl);
+                    $imagePath = UploadGameImage($file, ImageType::GameInGame);
+                    break;
```

Comment:
Looks like duplicated code may be starting to stack up here. I wonder if a private function may be beneficial?

```php
// untested pseudocode
private function processGameImage(string $dataUrl, string $imageType): string {
    $file = createFileArrayFromDataUrl($dataUrl);
    return UploadGameImage($file, $imageType);
}
```

Reply:
I thought about it, but it's just cutting two lines down to one for each case.

I've opted to use a mapping dictionary, similar to the one you proposed for the calling function.